src/DynamicData/Binding/SortAndBindOptions.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) 2011-2023 Roland Pheasant. All rights reserved.
+// Roland Pheasant licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for full license information.
+
+namespace DynamicData.Binding;
+
+/// <summary>
+/// Options for bind the bind and sort operators.
+/// </summary>
+public record struct SortAndBindOptions()
+{
+    /// <summary>
+    /// The reset threshold ie the number of changes before a reset is fired.
+    /// </summary>
+    public int ResetThreshold { get; init; } = BindingOptions.DefaultResetThreshold;
+
+    /// <summary>
+    /// When possible, should replace be used instead of remove and add.
+    /// </summary>
+    public bool UseReplaceForUpdates { get; init; } = BindingOptions.DefaultUseReplaceForUpdates;
+
+    /// <summary>
+    /// Should a reset be fired for a first time load
+    /// This option is due to historic reasons where a reset would be fired for the first time load regardless of the number of changes.
+    /// </summary>
+    public bool ResetOnFirstTimeLoad { get; init; } = BindingOptions.DefaultResetOnFirstTimeLoad;
```

Comment:
Do we need a historic setting on a new operator?  Or are you trying to make it easy for people who relied on the old behavior to migrate?

Reply:
In most cases you want a reset. Some user controls out there do not support reset, or maybe there's a requirement to notify individual item changes after bind. Only then is there a need to opt in.

Reply:
I thought about this one, and have gone with you suggestion.  Done and checked in.